components/calendar/test/calendar.visual-diff.js

```diff
@@ -389,7 +404,6 @@ describe('d2l-calendar', () => {
 	const tabToDates = async function() {
 		await page.keyboard.press('Tab');
 		await page.keyboard.press('Tab');
-		await page.keyboard.press('Tab');
```

Comment:
The `calendar.focus()` added in the `afterEach` block causes only 2 tabs to be needed